.github/workflows/release-package.yml

```
@@ -156,7 +156,7 @@ jobs:
     runs-on: ubuntu-20.04
     environment: publish
     needs: package
-    if: ${{ github.event.inputs.dry-run == 'false' }} 
+    if: ${{ inputs.dry-run == false }}
```

Renommer `dry-run` a `dry_run` et
```suggestion
    if: ${{ !inputs.dry_run }}
```